src/ts/localization/i18nApi.ts

```diff
@@ -0,0 +1,161 @@
+import { defaultBitmovinPlayerUiTranslations } from './i18n';
+
+interface StaticVocabulary {
```

Comment:
This should simply be called `Vocabulary`, because this is the only kind of vocabulary we have.